### ISSUE TITLE
Looks great!

### ISSUE THREAD

**Issue by joshullman** [ellismarte/FACQ#2]
Hey, I just remembered you made this and decided to come take a look.  Everything looks good!  Well organized and documented.  It might be helpful to add examples to each question.  For instance - 
Q: What are arguments?
A: Arguments are the values supplied to the command when it is called.
Ex: git [status], brew [install] <--- maybe bold or italicized?
A list of common arguments could be super useful as well.  Anyways, thanks for setting this up, I'm looking forward to using it in the future!
-Josh

**Comment by ellismarte** [comment]
That's a good point. I'll add it to the read me file. Feel welcome to clone, add/edit/delete, push, request a pull etc. 

Do you know if I can give people the ability to merge pull requests to this file?

**Comment by briankennedy1** [comment]
Yep, I think you can add collaborators. Here's the [GitHub doc](https://help.github.com/articles/adding-collaborators-to-a-personal-repository/) :octocat:
